apis/v1alpha2/httproute_types.go

```diff
@@ -457,6 +476,14 @@ type HTTPRouteMatch struct {
 	//
 	// +optional
 	ExtensionRef *LocalObjectReference `json:"extensionRef,omitempty"`
+
+	// Method specifies HTTP method matchers.
+	// If specified this matcher will only be picked if the request has the corresponding method
+	//
+	// Support: Core
+	//
+	// +optional
+	Method HTTPMethod `json:"method,omitempty"`
```

Comment:
I think this should be a pointer as it is an optional field.

Reply:
I don't understand the reasoning behind it, but you seem to be right. Every optional value which is not an array is a pointer.